libcxx/utils/ci/run-buildbot

```diff
@@ -217,7 +217,19 @@ function test-armv7m-picolibc() {
         "${@}"
 
     ${NINJA} -vC "${BUILD_DIR}/compiler-rt" install
-    mv "${BUILD_DIR}/install/lib/armv7m-none-unknown-eabi"/* "${BUILD_DIR}/install/lib"
+
+    # Prior to clang 19, armv7m-none-eabi normalised to armv7m-none-unknown-eabi.
+    # clang 19 changed this to armv7m-unknown-none-eabi. So for as long as 18.x
+    # is supported, we have to ask clang what the triple will be.
+    if [ ! -z "${CC}" ]
+    then
+        C_COMPILER=${CC};
+    else
+        C_COMPILER=cc;
+    fi
```

Comment:
Nits:
* Is there a reason to include the semicolons?
* I presume you have a reason for not writing `C_COMPILER=${CC-cc}`, or even just using `${CC-cc}` in situ?

Reply:
Because I am bad at bash :) Thanks for the tips I'll add those.